cloud/jenkins/psmdb_operator_gke_latest.groovy

```diff
@@ -228,12 +230,14 @@ void createCluster(String CLUSTER_SUFFIX) {
     withCredentials([string(credentialsId: 'GCP_PROJECT_ID', variable: 'GCP_PROJECT'), file(credentialsId: 'gcloud-key-file', variable: 'CLIENT_SECRET_FILE')]) {
         sh """
             export KUBECONFIG=/tmp/$CLUSTER_NAME-$CLUSTER_SUFFIX
-
             maxRetries=15
             exitCode=1
             while [[ \$exitCode != 0 && \$maxRetries > 0 ]]; do
                 gcloud container clusters create $CLUSTER_NAME-$CLUSTER_SUFFIX \
                     --release-channel $GKE_RELEASE_CHANNEL \
+                    --maintenance-window-start "00:00" \
```

Comment:
Lets make maintenance window +1 day from now.

Reply:
By specifying `"00:00"` as the start time and `"23:59"` as the end time, you are essentially allowing Google to perform maintenance at **any point throughout the entire day, every day**. That means practically to stay in maintenance forever. Isn't that what we actually want?

Reply:
I added
```
            gcloud container clusters update $CLUSTER_NAME-$CLUSTER_SUFFIX \
                --add-maintenance-exclusion-start "\$CURRENT_TIME" \
                --add-maintenance-exclusion-end "\$FUTURE_TIME"
```
where FUTURE_TIME is +6 hours (the life span of the cluster)